docs/contributing.rst

```diff
@@ -22,7 +22,7 @@ once:
 
    git clone https://github.com/letsencrypt/letsencrypt
    cd letsencrypt
-   ./bootstrap/install-deps.sh
+   ./letsencrypt-auto/letsencrypt-auto --os-packages-only
```

Comment:
Once we have `--no-autoupdate` or something, we should probably suggest it on this line?